docs/versioned_docs/version-zenith/quickstart/428201-custom-modules.mdx

```diff
@@ -30,6 +30,7 @@ To see this in action, create a custom Dagger module to build a Go binary, add i
 
     ```shell
     mkdir my-module
+    cd my-module
```

Comment:
Hm, alternatively, we could simplify this entire block to:

```
dagger init --sdk=go my-module
```

Reply:
Thank you. Just try and its work!

Fixed.